### ISSUE TITLE
Delete

### ISSUE THREAD

**Issue by rickardliljeberg** [OdeToCode/Memflex#41]
Is there any reason why there is no delete functionality in memflex, I am thinking that it would want to clear up oauth accounts and such and that it therefor would be beneficial with a delete method

**Comment by OdeToCode** [comment]
Yes, I can see that. To be honest, I don't remember why exactly I left it out, probably just wanted to defer it.
